scripts/ci/check_compliance.py

```diff
@@ -274,6 +274,9 @@ class ClangFormatCheck(ComplianceTest):
 
     def run(self):
         for file in get_files():
+            if Path(file).suffix not in ['.c', '.h']:
```

Comment:
.hpp and .cpp too, or do these use different formatting guidelines?

Reply:
I wasn't sure myself, if the `.clang-format` file can actually be applied to c++ files?